forte_wrapper/vader/sentiment_analysis.py

```diff
@@ -51,24 +53,44 @@ def __init__(self):
 
     def initialize(self, resources: Resources, configs: Config):
         # pylint: disable=unused-argument
```

Comment:
Are there `unused-argument` here? Looks like `reourses` and `configs` are used (too call the super)